app/assets/stylesheets/registration.scss

```diff
@@ -36,15 +36,18 @@
 	overflow: visible;
 }
 
-.ui-controlgroup > .ui-controlgroup-item {
+.ui-controlgroup>.ui-controlgroup-item {
 	float: left;
 	margin-left: 0;
 	margin-right: 0;
+	background: #fff 50% 50% repeat-x;
+	color: black;
 }
 
-.ui-state-active, .ui-widget-content .ui-state-active {
-	border: 1px solid #dc5541;
-	background: #fff 50% 50% repeat-x;
+.ui-state-active,
+.ui-widget-content .ui-state-active {
+	border: 1px solid #8c1515;
+	background: #8c1515 50% 50% repeat-x;
```

Comment:
do we need the position here?

Reply:
It was already there, I did not add it.

Reply:
but do we need it?

Reply:
I have no idea, a proper design that specified this would help.

Reply:
Doesn't seem to be necessary. But I think this is more a question for you: https://github.com/sul-dlss/argo/commit/af62dcdebd8d7b5ef1b1cb6e6dd2327bd9e14992